arch/x86/include/asm/barrier.h

```diff
@@ -87,9 +127,37 @@ do {									\
 
 #endif
 
+#else /* CONFIG_KTSAN */
+
+#define smp_store_release(p, v)						\
+do {									\
+	typeof(p) ___p1 = (p);						\
```

Comment:
add a TODO that this must be replaced with ktsan_atomic_store(p, v, ktsan_memory_order_release)